### PR TITLE
CNS-565: query all available geolocations

### PR DESCRIPTION
**Pull request by oren-lava**
Created a query for the plans module that prints all supported geolocations and their uint value (in hex) to ease policy configuration

### CODE REVIEW

**Review by orenl-lava — DISMISSED** [review]

File: proto/plans/query.proto [hunk 1]
```diff
@@ -61,5 +61,4 @@ message QueryInfoResponse {
   Plan plan_info = 1 [(gogoproto.nullable) = false];
 }
 
-
```
[the remark this answers is not in the document]
Comment (oren-lava) [reply proto/plans/query.proto:64]: yep.

File: x/pairing/keeper/staking.go [hunk 2]
```diff
@@ -60,6 +60,7 @@ func (k Keeper) StakeNewEntry(ctx sdk.Context, creator string, chainID string, a
 	if !planstypes.IsValidGeoEnum(int32(geolocation)) {
 		return utils.LavaFormatWarning("can't register for no geolocation or geolocation outside zones", fmt.Errorf("invalid geolocation"),
```
[the remark this answers is not in the document]
Comment (oren-lava) [reply x/pairing/keeper/staking.go]: done. see https://github.com/lavanet/lava/commit/28d5ae7db60c2e06b24938205101cc4e94377d15

File: x/plans/types/policy.go [hunk 3]
```diff
@@ -218,7 +218,7 @@ func parsePolicyEnumValue(enumType interface{}, strVal string) (interface{}, err
 	case uint64:
 		geo, err := ParseGeoEnum(strVal)
 		if err != nil {
-			return 0, fmt.Errorf("invalid geolocation %s", strVal)
+			return 0, fmt.Errorf("invalid geolocation %s. Valid geolocations: %s", strVal, PrintGeolocations())
```
[the remark this answers is not in the document]
Comment (oren-lava) [reply x/plans/types/policy.go]: done. see https://github.com/lavanet/lava/commit/28d5ae7db60c2e06b24938205101cc4e94377d15

File: x/plans/types/geolocation.go [hunk 4]
```diff
@@ -95,3 +96,33 @@ func (g *Geolocation) UnmarshalJSON(b []byte) error {
...
+	sort.Slice(geos, func(i, j int) bool {
+		return geos[i].val < geos[j].val
+	})
+
+	var geosStr string
+	for _, info := range geos {
+		geosStr += info.String() + ", "
+	}
+
+	return geosStr[:len(geosStr)-2]
```
[the remark this answers is not in the document]
Comment (orenl-lava) [reply x/plans/types/geolocation.go]: nice!

**Review by Yaroms — APPROVED** [review]

**Pull request merged**